common/data/rumble.json

```diff
@@ -53987,6 +53987,431 @@
         "criteria": "near"
       }
     },
+    {
+      "ability": [
+        {
+          "effects": [
+            {
+              "attributes": [ "ATK" ],
+              "effect": "buff",
+              "level": 1,
+              "targeting": { "targets": [ "INT" ] }
```

Comment:
the current pattern for types only is to wrap them in [], so "[INT]". It's a holdover from the way things used to be done. There's some exception handling so it won't break things, but it's good to be consistent. Again, this is for types only.